### PR TITLE
1014 es l10n fixes

### LINKED ISSUE #1014: (Spanish) Localization issues
There are some issues with the Spanish translation:
1-Recently Used is not correctly translated. It should be `recientemente`
IMG_235F4C80B424-1

2- When entries are sorted by Recent, the name in the button is shown with ellipsis in the middle, it does not look nice :/ 
IMG_3CFB2607AD3D-1

### PR DESCRIPTION
**Pull request by sashei**
Fixes #1014

## Testing and Review Notes
The one issue that I couldn't fix was the incorrect gender for the `copiado/a`... We interpolate the string for `username` and `password` irrespective of gender, so I think making a larger change to introduce separate strings for `Username Copied` and `Password Copied` is out of scope for now.

## To Do

- [x] double check the original issue to confirm it is fully satisfied
- [x] add testing notes and screenshots in PR description to help guide reviewers
- [x] make sure CI builds are passing (e.g.: fix lint and other errors)

### CODE REVIEW

File: lockbox-ios/Common/Resources/Strings/es.lproj/Localizable.strings [hunk 1]
```diff
@@ -101,7 +101,7 @@
 "recent" = "Reciente";
 
 /* Label for the option sheet action allowing users to sort the logins list by the most recently used logins */
-"recently_used" = "Usado cecientemente";
+"recently_used" = "Usado recientemente";
```
Comment (joeyg) [inline lockbox-ios/Common/Resources/Strings/es.lproj/Localizable.strings]: Won't this get overwritten during the next import?

Comment (devinreams) [reply lockbox-ios/Common/Resources/Strings/es.lproj/Localizable.strings]: @joeyg yes, I'll go submit a change to the upstream Pontoon service:
https://pontoon.mozilla.org/es/lockwise-ios/lockwise-ios.xliff/?search=recent&string=199215

Comment (sashei) [reply lockbox-ios/Common/Resources/Strings/es.lproj/Localizable.strings]: goood point.

**Comment by sashei** [comment]
@devinreams @Pike what is the process for uplifting reported string errors to Pontoon?

**Comment by devinreams** [comment]
image

I just looked at the report from Isabel. Any others or just this one?

**Comment by sashei** [comment]
@devinreams well, see my comment above about doing the `Username / Password copied` substitution in gendered languages >.<

**Comment by sashei** [comment]
also @isabelrios I see a localized FxA login in French, just not in Spanish. Not sure what the story is there.

**Comment by devinreams** [comment]
>see my comment above about doing the Username / Password copied substitution

Ah yes, I agreed with you it would be out of scope and require redoing/breaking strings so I'll file this follow-up: #1028 

> see a localized FxA login in French, just not in Spanish. Not sure what the story is there.

I also see a German localized page. So this seems like an FxA language detection bug perhaps...

**Comment by Pike** [comment]
Leaving the suggestion in pontoon is the right way to do it, thanks.

**Pull request merged**